docs/checkit.js

```diff
@@ -777,55 +786,118 @@ CheckIt.prototype.validateInput = function ($form, $inputFormGroup, inputId) {
 
 CheckIt.prototype.validateForm = function (startDateString, endDateString) {
     "use strict";
+    // First check that input was actually given for title, startDate
+    // and endDate
     var validateTitle = this.validateInput(this.$fullForm, this.$titleFormGroup, 'calendarTitle'),
         
         validateStartDate = this.validateInput(this.$fullForm, this.$startDateFormGroup, 'startDate'),
         
         validateEndDate = this.validateInput(this.$fullForm, this.$endDateFormGroup, 'endDate'),
     
         isValid = validateTitle && validateStartDate && validateEndDate;
-    
+
     // Make sure input is OK before parsing and validating dates
     return (isValid && this.validateDates(startDateString, endDateString, this.$endDateFormGroup));
    
 };
 
-CheckIt.prototype.createCalendar = function () {
-    // Create a new calendar and display it.
+CheckIt.prototype.createCalendar = function (calState) {
+    // creates a calendar
+    
     "use strict";
-    var title = this.$calendarTitle.val(),
-        start = this.$startDate.val(),
-        end = this.$endDate.val(),
-        state,
-        calendar,
+    var calendar,
         initP,
         buildP;
-        
+    //if (this.validateForm(start, end)) {
```

Comment:
I have no idea.

Reply:
Oh wait, I know. Because when you click on "create calendar" on the website, the function editOrCreate gets run, and in there the form is validated and based on which button was pressed either editCalendar gets run or createCalendar.